games-emulation/retroarch/retroarch-1.7.1.recipe

```diff
@@ -73,7 +72,7 @@ PATCH()
 
 BUILD()
 {
-	./configure --prefix="$prefix" --global-config-dir="$settingsDir" \
+	./configure --prefix="$prefix" --global-config-dir="$sysConfDir" \
```

Comment:
```
Warning: POLICY ERROR: Invalid top-level package entry "etc"
Warning: POLICY ERROR: Package declares non-existent global writable file "settings/retroarch.cfg" as included
Warning: Error: packaging policy violation(s) in strict mode
Error: Build has failed - stopping.
```
Let me check if this was caused by this recent change.

Reply:
```diff
-sysConfDir
+sysconfDir
```
(`s/C/c/`)

Reply:
FWIW:
```
cd games-emulation/retroarch/work-1.7.1
grep -E '^settingsDir=|^sysconfDir=' wrapper-script 
settingsDir='/packages/retroarch-1.7.1-1/.self/settings'
sysconfDir='/packages/retroarch-1.7.1-1/.settings'
```
(I haven't figured yet if there are any differences. Both work fine.)

Reply:
My guess is that it's caused by this

+GLOBAL_WRITABLE_FILES="
+	$relativeSettingsDir/retroarch.cfg auto-merge
+	"